packages/e2e/scripts/playwright/axe-clean.spec.ts

```diff
@@ -65,6 +64,11 @@ const testsToSkip = [
 const testsWithCustomDelay = {
   'components-subdomainnavbar--mobile-menu-open': 5000, // takes a while for the menu to open
 }
+const ignoreViolations = [
+  'landmark-one-main', // on most of the sotries we don't have a main landmark
+  'page-has-heading-one', // on some stories we dont have a heading,
```

Comment:
In some components, we do need to test for this, like Hero 🤔... can we create a safe list for those stories?

Reply:
@rezrah you are absolutely right! We can expand on this 100% and then have a list on which stories we need the check "page-has-heading-one".

We can have it as an object, since the filter would be much faster then. Something like this:
```javascript
const ignoreViolations = {
  'landmark-one-main':  {except: []}, // or {} or {except: null | false | []} 
  'page-has-heading-one': {except: [...]}
  'region': {except: []}
}
```